amazon_transcribe/utils.py

```diff
@@ -38,3 +43,19 @@ def ensure_boolean(val):
         return val
     else:
         return val.lower() == "true"
+
+async def apply_realtime_delay(
+    stream: AudioStream,
+    reader: aiofile.Reader,
+    bytes_per_sample: int,
+    sample_rate: float,
+    channel_nums: int,
+) -> None:
```

Comment:
Let's make sure to add a brief docstring so it's clear outside of the example code what this is used with.